cache/compressed_secondary_cache_test.cc

```diff
@@ -800,21 +797,21 @@ Cache::CacheItemHelper CompressedSecondaryCacheTest::helper_fail_(
     CompressedSecondaryCacheTest::DeletionCallback);
 
 TEST_F(CompressedSecondaryCacheTest, BasicTestWithNoCompression) {
-  BasicTest(false, false);
+  BasicTest(/*sec_cache_is_compressed=*/false, /*use_jemalloc=*/false);
```

Comment:
Can some of the tests in this file be parameterized? That way, the custom_split_and_merge can be a parameter in order to increase coverage.

Reply:
Addressed in the latest commit.